jetson/odrive_bridge/src/__main__.py

```diff
@@ -116,74 +116,74 @@ def __str__(self):
         return self.__class__.__name__
 
 
-class DisconnectedState(State):
-    def on_event(self, event):
+class Disconnected_State(State):
+    def state_on_event(self, odrive_event):
         """
         Handle events that are delegated to the Disconnected State.
         """
         global modrive
-        if (event == Event.ARM_CMD):
+        if (odrive_event == Odrive_Event.ARM_CMD):
             modrive.disarm()
             modrive.reset_watchdog()
             modrive.arm()
-            return ArmedState()
+            return Armed_State()
 
         return self
 
 
-class DisarmedState(State):
-    def on_event(self, event):
+class Disarmed_State(State):
+    def state_on_event(self, odrive_event):
         """
         Handle events that are delegated to the Disarmed State.
         """
         global modrive
-        if (event == Event.DISCONNECTED_ODRIVE):
-            return DisconnectedState()
+        if (odrive_event == Odrive_Event.DISCONNECTED_ODRIVE):
+            return Disconnected_State()
 
-        elif (event == Event.ARM_CMD):
+        elif (odrive_event == Odrive_Event.ARM_CMD):
             modrive.arm()
-            return ArmedState()
+            return Armed_State()
 
-        elif (event == Event.ODRIVE_ERROR):
-            return ErrorState()
+        elif (odrive_event == Odrive_Event.ODRIVE_ERROR):
+            return Error_State()
 
         return self
 
 
-class ArmedState(State):
-    def on_event(self, event):
+class Armed_State(State):
+    def state_on_event(self, odrive_event):
         """
         Handle events that are delegated to the Armed State.
         """
         global modrive
 
-        if (event == Event.DISARM_CMD):
+        if (odrive_event == Odrive_Event.DISARM_CMD):
             modrive.disarm()
-            return DisarmedState()
+            return Disarmed_State()
 
-        elif (event == Event.DISCONNECTED_ODRIVE):
-            return DisconnectedState()
+        elif (odrive_event == Odrive_Event.DISCONNECTED_ODRIVE):
+            return Disconnected_State()
 
-        elif (event == Event.ODRIVE_ERROR):
-            return ErrorState()
+        elif (odrive_event == Odrive_Event.ODRIVE_ERROR):
+            return Error_State()
 
         return self
 
 
-class ErrorState(State):
-    def on_event(self, event):
+class Error_State(State):
+    def state_on_event(self, odrive_event):
         """
         Handle events that are delegated to the Error State.
         """
         global modrive
         print(dump_errors(modrive.odrive, True))
-        if (event == Event.ODRIVE_ERROR):
+        if (odrive_event == Odrive_Event.ODRIVE_ERROR):
             try:
                 modrive.reboot()  # only runs after initial pairing
             except Exception as e:
                 print('Exception caught as:', e)

```

Comment:
this isn't current?? it shouldn't be printing the exceptions anymore

Reply:
I branched it from travis-free (and the code is also in main) so something must've changed somewhere or it always printed exceptions and that change never went through

Reply:
hm maybe it's just different in auton-integration then, prob better to not always print tbh fills up the terminal